### PR TITLE
align intro & service buttons

### PR DESCRIPTION
**Pull request by DanBuda11**
Issue #91 - align all buttons.

Combo of flexbox and position: absolute to align both service & intro buttons.

### CODE REVIEW

File: app/src/sass/_buttons.scss [hunk 1]
```diff
@@ -71,3 +71,13 @@ button::-moz-focus-inner {
...
 }
+
+@mixin button-flex-container {
+  display: flex;
+  justify-content: center;
+}
+
+@mixin button-flex {
+  position: absolute;
+  bottom: 70px;
```
Comment (mateoclarke) [inline app/src/sass/_buttons.scss]: Thank you for these mixins. 👌

File: app/package.json [hunk 2]
```diff
@@ -39,6 +39,6 @@
     "lint": "eslint src",
     "nibble": "eslint-nibble src",
     "predeploy": "npm run build",
-    "deploy" : "gh-pages -d build"
+    "deploy": "gh-pages -d build"
```
Comment (mateoclarke) [inline app/package.json]: 🕵️  watch a catch

**Comment by mateoclarke** [comment]
Hey @DanBuda11, after diving into the CSS a bit, what did you think about the way it is architected? Be honest. Was it easy to understand and make contributions to? Or would you recommend discussing new approaches. I've been hearing a lot about [styled-components](https://www.styled-components.com/docs/basics#motivation) but it seemed like an extra learning curve to climb. 

Seriously, thanks for your contribution!

**Comment by DanBuda11** [comment]
I'll take a deeper look at it but it made sense to me. I did have to spend a little time figuring out what was where but that's probably because it was the first time I'd looked at it. Glad to help!

**Pull request merged**